linera-views/src/backends/journaling.rs

```diff
@@ -237,6 +244,9 @@ where
         if Self::is_fastpath_feasible(&batch) {
             self.store.write_batch(batch).await
         } else {
+            if !self.has_exclusive_access {
+                return Err(JournalConsistencyError::JournalRequiresExclusiveAccess.into());
+            }
```

Comment:
The action is indeed necessary for the system to work correctly.
However, I would say that this kills the interest of journaling.

The problem is that what is stored as blobs is large data like Wasm compiled code. Given the limits on size for the fastpath this could be a problem. Still, I have not been able to enable the problem.

Right now, I do not see how to address this issue and have journal and non-exclusive access.